Formula/goaccess.rb

```diff
@@ -14,10 +13,12 @@ class Goaccess < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "gettext"
```

Comment:
This used to work with system gettext, what changed?

Reply:
Not sure. It couldn't find the `libintl.h` dependency. Maybe could try specifying directly. I was following the previous PR which was closed because they couldn't get the compile to work.